### PR TITLE
Update SETUP.md with troubleshooting for error "NU1403: Package content hash validation failed for ..."

### PR DESCRIPTION
**Pull request by fedemkr**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other

## Objective

Added troubleshooting for error "NU1403: Package content hash validation failed for ..." on Setup.md

## Code changes

* **Setup.md:** Added troubleshooting for error "NU1403: Package content hash validation failed for ..."

## Before you submit
- [ ] I have checked for formatting errors (`dotnet tool run dotnet-format --check`) (required)
- [ ] If making database changes - I have also updated Entity Framework queries and/or migrations
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -152,6 +152,17 @@ dotnet run
...
+### Troubleshooting
+
+If you get an error while restoring the nuget packages like `error NU1403: Package content hash validation failed for ...` then these following commands should fix the problem:
+```
+dotnet nuget locals all --clear
+git clean -xfd
+git rm **/packages.lock.json -f
+dotnet restore
+```
+The thing is that on differnet OS the hash is done differently so if you've downloaded the `packages.lock.json` files from git and they were created on a different OS then you might run into this issue, and the previous commands should clear everything, remove the `packages.lock.json` files and create them again on the restoring. ([source](http://thecodemanual.pl/2020/03/11/caching-not-only-nuget-packages-on-azure-devops.html))
```
Comment (Hinton) [inline SETUP.md:164]: We already have a troubleshooting section below, lets move this there as well to keep things consistent.

Comment (fedemkr) [reply SETUP.md:164]: ohh sorry, for some reason I thought that troubleshooting section was referring to testing only. I'll move it

File: SETUP.md [hunk 2]
```diff
@@ -164,6 +164,14 @@ From within Rider, launch both the Api project and the Identity project by click
...
 ### Troubleshooting
 * If you get a 404 error, the projects may be listening on a non-default port. Check the output of your running projects to check the port they are listening on.
+* If you get an error while restoring the nuget packages like `error NU1403: Package content hash validation failed for ...` then these following commands should fix the problem:
+```
+dotnet nuget locals all --clear
+git clean -xfd
+git rm **/packages.lock.json -f
+dotnet restore
+```
+The thing is that on differnet OS the hash is done differently so if you've downloaded the `packages.lock.json` files from git and they were created on a different OS then you might run into this issue, and the previous commands should clear everything, remove the `packages.lock.json` files and create them again on the restoring. ([source](http://thecodemanual.pl/2020/03/11/caching-not-only-nuget-packages-on-azure-devops.html))
```
Comment (Hinton) [inline SETUP.md:174]: We can indent the lines below 167 which will put the content inside the list.

```suggestion
* If you get an error while restoring the nuget packages like `error NU1403: Package content hash validation failed for ...` then these following commands should fix the problem:
  ```dotnet nuget locals all --clear
  git clean -xfd
  git rm **/packages.lock.json -f
  dotnet restore```
  For more details read https://github.com/NuGet/Home/issues/7921#issuecomment-478152479
```

**Review by Hinton — COMMENTED** [review]
I wonder if this could be solved by using `<DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>` per https://github.com/NuGet/Home/issues/7921#issuecomment-478152479

**Comment by fedemkr** [comment]
Yes @Hinton , that could potentially fix that but I read several comments from people that `DisableImplicitNuGetFallbackFolder` didn't work on their machines. But yes, we could add that to the readme as another potential fix for the problem. I personally didn't test it but other people claimed that it did the trick.

**Review by Hinton — APPROVED** [review]
LGTM, thanks for adding this to the documentation.

**Pull request merged**